src/libstd/process.rs

```diff
@@ -828,13 +828,33 @@ impl Child {
 /// this function at a known point where there are no more destructors left
 /// to run.
 ///
+/// ## Platform-specific behavior
+///
+/// Despite `code` being an `i32`, some platforms take into account only a
+/// certain range of these bits. On macOS and Linux, `exit`'s underlying system
+/// function only considers the lower eight bits of `code`. For example, on
+/// these platforms, calling `exit` with values `0x00_00` and `0x0f_00` will be
+/// handled identically. Other platforms, like Windows, will take into account
+/// all 32 bits of `code`.
```

Comment:
It looks like your opengroup link says `exit` only takes into consideration the least significant eight bits. I have hesitations about *just* citing that as it doesn't match the behavior on Windows which takes into account *all* 32 bits with no truncation (as per [this comment](https://github.com/rust-lang/rust/issues/35046#issuecomment-235484403) by @retep998).